kani-driver/src/call_cbmc.rs

```diff
@@ -310,13 +340,30 @@ impl VerificationResult {
 }
 
 /// We decide if verification succeeded based on properties, not (typically) on exit code
-fn determine_status_from_properties(properties: &[Property]) -> VerificationStatus {
-    let number_failed_properties =
-        properties.iter().filter(|prop| prop.status == CheckStatus::Failure).count();
-    if number_failed_properties == 0 {
-        VerificationStatus::Success
+fn verification_outcome_from_properties(
+    properties: &[Property],
+    should_panic: bool,
+) -> (VerificationStatus, Option<PanicOutcome>) {
+    let failed_properties: Vec<&Property> =
+        properties.iter().filter(|prop| prop.status == CheckStatus::Failure).collect();
```

Comment:
Do I need to filter other properties here? If I'm not mistaken, `cover` properties could interfere...

Reply:
Actually, `cover` properties will be filtered out since status can only be `SATISFIABLE` or `UNSATISFIABLE`.